tests/LivewireDatatableQueryBuilderTest.php

```diff
@@ -41,6 +41,10 @@ public function it_creates_a_query_builder_for_base_columns()
         $subject->sort(1);
 
         $this->assertEquals('select "dummy_models"."id" as "id", "dummy_models"."subject" as "subject" from "dummy_models" order by `subject` asc, `id` desc', $subject->getQuery()->toSql());
+
```

Comment:
Should make a new test class for multisortable. These will live here temporarily